### DISCUSSION TITLE
Namespaced Keda installation

### DISCUSSION THREAD

**Discussion by 1277kii** [kedacore/keda-docs#400] [General]
Hi,
I was unable to find information if I can make namespaced KEDA installation (if CRDs and ClusterRoles are installed by the k8s administrator)?
I am asking as I have access only to a specific namespace and I am not admin of the cluster.

Best regards,
 Kaloyan Iliev

**Comment by zroubalik** [comment]
This was briefly discussed here: https://github.com/kedacore/charts/issues/129

But bear in mind, that in order to install Metrics Server (and scaling 1<->N) you need some admin permissions
